pkg/skaffold/initializer/deploy/kubectl_test.go

```diff
@@ -47,7 +47,8 @@ spec:
 			},
 		},
 	}
-	testutil.CheckDeepEqual(t, expectedConfig, k.DeployConfig())
+	deployConfig, _ := k.DeployConfig()
```

Comment:
We should be checking the profiles is nil, no?

Reply:
yeah i suppose you're right. i'll update